NDcPP_v2_2e.adoc

```diff
@@ -2806,36 +2809,116 @@ _The preferred method for verification is the Subject Alternative Name using DNS
 
 Finally, the client should avoid constructing reference identifiers using wildcards. However, if the presented identifiers include wildcards and the TOE supports wildcard, the client must follow the best practices regarding matching; these best practices are captured in the evaluation activity. The exception being, the use of wildcards is not supported when using IP address as the reference identifier
 
-*FCS_TLSC_EXT.1.3* When establishing a trusted channel, by default the TSF shall not establish a trusted channel if the server certificate is invalid. The TSF shall also [selection:
+*FCS_TLSC_EXT.1.3* The TSF shall not establish a trusted channel if the server certificate is invalid [selection:
 
-* _Not implement any administrator override mechanism_
-* _require administrator authorization to establish the connection if the TSF fails to_ [selection: _match the reference identifier, validate certificate path, validate expiration date, determine the revocation status] of the presented server certificate_
+* _without any administrator override mechanism._
+* _except with the following administrator override:   If the TSF fails to [selection: match the reference identifier, validate certificate path, validate expiration date, determine the revocation status] the TSF shall allow the administrator to provide override authorization to establish the connection on a per certificate basis._
 
 ].
 
 *_Application Note {counter:appnote_count}_*
 
+_‘Revocation status’ refers to an OCSP or CRL response that indicates the presented certificate is invalid. Inability to make a connection to determine validity shall be handled as specified in FIA_X509_EXT.2.2. If the revocation status of a certificate received by the TOE is ambiguous (e.g. ‘unknown’), this should be treated similar to the situation where no connection could be established to the revocation server and the option ‘determine the revocation status’ could be chosen for this._
+
+_The purpose of the explicit selection in the SFR is to prevent the TOE providing an authorization mechanism for situations other than specified in the selection (e.g. one or more certificates in the certification path have been revoked and this status is known to the TOE). Override may be sought or granted at any time, though this typically occurs when an invalid certificate is presented during connection setup.  Override decisions may be stored and then consulted later, to permit connections_ _using these otherwise-invalid certificates to establish trusted channels without user or administrator action._
```

Comment:
The word 'override' wasn't included in the first sentence. Should it be added?

Reply:
Added "override" and removed "mechanism".